bedrock/lightbeam/templates/lightbeam/lightbeam.html

```diff
@@ -107,15 +101,6 @@ <h2 class="page-slogan">{{_('How Lightbeam Works')}}</h2>
       </p>
     </figcaption>
   </figure>
-  <figure class="span4 contribute">
-    <img src="{{ static('img/lightbeam/Lightbeam_Illustration3.png') }}" alt="">
```

Comment:
Can you please remove this image from the repo if it's no longer being used? Thanks